src/frontend/components/isbn-registry/identifierRanges/isbn/Batch.jsx

```diff
@@ -160,7 +168,7 @@ function Batch({identifierBatch, intl, history, downloadBatch, deleteBatch}) {
       </div>
 
       {/* Show loading spinner while deleting batch */}
-      <Backdrop open={loading} onClick={handleCloseBackdrop}>
+      <Backdrop open={loading && showSpinner} onClick={handleCloseBackdrop}>
```

Comment:
showSpinner does not need to be introduced as new prop/variable. State of "loading" should reflect whether there is loading ongoing or not. Since deleteBatch function is async function that is called in handleDeleteBatch, you may change handleDeleteBatch to be also an async function and use await keyword during calling deleteBatch. If you introduce setLoading(false) to the end of this function, the state of loading is automatically updated whatever the result of the deletion is.

Reply:
Actually, noticed that this leads to a bug when the other modal is showing so it's the preferred implementation. :) Resolved